### PR TITLE
WIP: KillSwitch: Add support for connman

### PR DESCRIPTION
**Pull request by infirit**
**Untested** (tested now, works ok so far) work in progress but this is what I am thinking of to add connmans support.

>Connman blocks the rfkill device for writing and we need to use it to
>set the power state.

### CODE REVIEW

File: blueman/plugins/applet/KillSwitch.py [hunk 1]
```diff
@@ -126,7 +149,13 @@ def reply(*_):
         def error(*_):
             cb(False)
 
-        Mechanism().SetRfkillState(str('(b)'), state, result_handler=reply, error_handler=error)
+        if self._have_connman:
```
Comment (cschramm) [inline blueman/plugins/applet/KillSwitch.py]: You could just check for `self._connman_proxy` and save that other variable.

**Comment by progandy** [comment]
That seems to work fine.

File: blueman/plugins/applet/KillSwitch.py (hunk 1 above)
Comment (infirit) [inline blueman/plugins/applet/KillSwitch.py]: Good one, will do that before merging 👍

**Comment by infirit** [comment]
Thanks for testing @progandy, merged and pushed to master.

**Pull request merged**